data/tabular/chemcaption_rdkit/meta.yaml

```diff
@@ -0,0 +1,349 @@
+---
+name: chemcaption_rdkit
+description: |-
+    This dataset contains molecular descriptors, mostly derived using RDKit.
+targets:
+    - id: num_valence_electrons
+      type: categorical
+      description: number of valence electrons
+      names:
+          - noun: number of valence electrons
+          - noun: valence electron count
+          - only_name: valence electrons
+    - id: rotable_proportion
+      type: continuous
+      significant_digits: 3
+      description: proportion of rotatable bonds
+      names:
+          - noun: proportion of rotatable bonds
+          - noun: rotatable bond proportion
+    - id: non_rotable_proportion
+      type: continuous
+      significant_digits: 3
+      description: proportion of non-rotatable bonds
+      names:
+          - noun: proportion of non-rotatable bonds
+          - noun: non-rotatable bond proportion
+    - id: num_single_bonds
+      type: continuous
+      significant_digits: 3
+      description: number of single bonds
+      names:
+          - noun: fraction of single bonds
+          - noun: single bond fraction
+    - id: num_double_bonds
+      type: continuous
+      significant_digits: 3
+      description: number of double bonds
+      names:
+          - noun: fraction of double bonds
+          - noun: double bond fraction
+          - noun: double bond proportion
+    - id: num_triple_bonds
+      type: continuous
+      significant_digits: 3
+      description: number of triple bonds
+      names:
+          - noun: fraction of triple bonds
+          - noun: triple bond fraction
+          - noun: triple bond proportion
+    - id: num_aromatic_bonds
+      type: continuous
+      significant_digits: 3
```

Comment:
same here, can this be a float number?

Reply:
added now also a check that this is a float